### DISCUSSION TITLE
阿里云：failed link: failed get link: UserNotAllowedAccessResource:user not allowed access resource for violation

### DISCUSSION THREAD

**Discussion by frick007** [AlistGo/alist#5572] [General]
下载文件时显示标题错误，请教各位如何解决？

**Comment by mgz0227** [comment]
我也遇到了这个情况

**Comment by anwen-anyi** [comment]
![image](https://github.com/alist-org/alist/assets/56105412/c36d8bf7-18f8-41fa-bc54-8feba5898505)

近期阿里云盘升级了新的安全策略，在使用阿里云盘open时遇到上图所示错误👆

```
failed link: failed get link: UserNotAllowedAccessResource:user not allowed access resource for violation
```

以上错误码表示账号已经封禁，不给予解封，🔔已经明确告知禁止使用阿里云盘搭建包括但不限于 图床、软件床 和 视频外链到视频网站播放 等分发服务以及一些没有提到的不规范使用的方式，若违反被冻结/封号后果自负，并无法解除

但是本次升级优化之后被封禁的用户还可以登录与访问，但不能下载和在线播放，可以通过快传文件给自己小号的方式转移重要文件🫶（以前都是直接无法登录）


‼️再次提醒，请勿将阿里云盘open与他人共享使用，否则账号封禁后果自负，请勿抱有侥幸心理‼️

为了您的账号安全 / 防止被爬虫 / 其它人恶意访问等不是本人使用，或者您的AList不公开请开启如下选项（自行选择是否开启）：
1️⃣AList后台 -> 用户 -> guest -> 停用（默认为停用状态）
2️⃣AList后台 -> 设置 -> 全局 -> 全部签名
3️⃣元信息 -> 路径选择 选择 / -> 设置密码 -> 勾选后面的应用到子文件夹
如果不分享，停用访客账户后建议再开启一道保险 启用全部签名选项

🔆如果您未公开AList如何检查是否被爬虫/其它人访问如何查看：
AList应用程序同级目录下的 /data/log/log.log 日志文件中 时间、路径、IP等选项即可查看

Q：至于具体多少个不同IP、多少流量会被判定为共享？
A：不知道具体多少个你只要不分享你自己没有被爬虫/其它人访问肯定没问题，建议自己去日志自查一下是否被爬虫或者没有防护措施在被人使用，你要是一下五六七八台不同IP同时访问，那肯定会判定为共享，还有就是使用多少流量不知道具体的，都是通过安全机制自动执行的。

⚠️再次提醒规范使用，若违反被冻结/封号后果自负，并无法解除， 没有对外开放的自查一下AList日志是否有被他人、爬虫在访问。

**Comment by ArcherLiong** [comment]
好家伙，我就是海报刮削，这也被封了

**Reply by anwen-anyi** [reply]
> 好家伙，我就是海报刮削，这也被封了
> 
> 

不允许刮 刮海报2年前就这样

**Reply by ArcherLiong** [reply]
之前阿里云非open驱动的时候感觉还是很安全的。算了。。之前几十部几十部的刮是很危险。。毕竟为了plex和jellyfin使用嘛。接下来我得珍惜剩下的号了。。。

**Comment by BlueSkyXN** [comment]
恭喜封号乐

**Comment by ITManCHINA** [comment]
草了，今天才发现我号也没了淦

**Comment by ArcherLiong** [comment]
不用担心，最近阿里云已经开始限速了。账号价值越来越低了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月07日 00:40 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [alist-org/alist] 阿里云：failed link: failed get link: UserNotAllowedAccessResource:user not allowed access resource for violation (Discussion #5572) |

草了，今天才发现我号也没了淦

**Reply by ITManCHINA** [reply]
也是，最近把资源转移一下准备跑路咯，可惜了内测送的3TB
